apps/web/tests/unit/settings-preferences.test.ts

```
@@ -50,6 +56,18 @@ describe("page tab color preferences", () => {
     expect(normalizePageTabColors(null)).toBe("follow-sidebar");
   });
 
+  it("preserves a fixed header after the new default migration is current", () => {
+    localStorage.setItem("veolms-sidebar-header-default-version", "inline-v1");
+    localStorage.setItem(
+      "veolms-sidebar-preferences",
+      JSON.stringify({ headerLayout: "fixed" }),
+    );
+
+    window.eval(getSurfaceDepthBootstrapScript());
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)